README.md

```diff
@@ -1,583 +1,53 @@
-  namespace: default
-  resourceVersion: "1881"
-  selfLink: /apis/kubeflow.org/v1alpha1/namespaces/default/tfjobs/example-job
-  uid: e11f9577-b5e5-11e7-8522-42010a8e01a4
-spec:
-  RuntimeId: 76no
-  replicaSpecs:
-  - replicas: 1
-    template:
-      metadata:
-        creationTimestamp: null
-      spec:
-        containers:
-        - image: gcr.io/tf-on-k8s-dogfood/tf_sample:dc944ff
-          name: tensorflow
-          resources: {}
-        restartPolicy: OnFailure
-    tfPort: 2222
-    tfReplicaType: MASTER
-  - replicas: 1
-    template:
-      metadata:
-        creationTimestamp: null
-      spec:
-        containers:
-        - image: gcr.io/tf-on-k8s-dogfood/tf_sample:dc944ff
-          name: tensorflow
-          resources: {}
-        restartPolicy: OnFailure
-    tfPort: 2222
-    tfReplicaType: WORKER
-  - replicas: 2
-    template:
-      metadata:
-        creationTimestamp: null
-      spec:
-        containers:
-        - image: tensorflow/tensorflow:1.3.0
-          name: tensorflow
-          resources: {}
-          volumeMounts:
-          - mountPath: /ps-server
-            name: ps-config-volume
-        restartPolicy: OnFailure
-    tfPort: 2222
-    tfReplicaType: PS
-  tensorboard:
-    logDir: /tmp/tensorflow
-    serviceType: ""
-    volumeMounts: null
-    volumes: null
-  tfImage: tensorflow/tensorflow:1.3.0
-status:
-  conditions: null
-  controlPaused: false
-  phase: Done
-  reason: ""
-  replicaStatuses:
-  - ReplicasStates:
-      Succeeded: 1
-    state: Succeeded
-    tf_replica_type: MASTER
-  - ReplicasStates:
-      Running: 1
-    state: Running
-    tf_replica_type: WORKER
-  - ReplicasStates:
-      Running: 2
-    state: Running
-    tf_replica_type: PS
-  state: Succeeded
-
-```
-
-The first thing to note is the **RuntimeId**. This is a random unique
-string which is used to give names to all the K8s resouces
-(e.g Job controllers & services) that are created by the TFJob.
-
-As with other K8s resources status provides information about the state
-of the resource.
-
-**phase** - Indicates the phase of a job and will be one of
- - Creating
- - Running
- - CleanUp
- - Failed
- - Done
-
-**state** - Provides the overall status of the job and will be one of
-  - Running
-  - Succeeded
-  - Failed
-
-For each replica type in the job, there will be a ReplicaStatus that
-provides the number of replicas of that type in each state.
-
-For each replica type, the job creates a set of K8s
-[Job Controllers](https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/)
-named
-
-```
-${REPLICA-TYPE}-${RUNTIME_ID}-${INDEX}
-```
-
-For example, if you have 2 parameter servers and runtime id 76n0 TFJob
-will create the jobs
-
-```
-ps-76no-0
-ps-76no-1
-```
-
-### TensorFlow Logs
+## Getting Started
 
-Logging follows standard K8s logging practices.
+You can easily setup the environment, You could read [the quick start to start a tour of TFJob operator](docs/get_started.md).
```

Comment:
`You` should be lowercase

Also maybe a rewording suggestion? `Setting up the environment is easy.  Start by reading the quick start guide for a tour of the TFJob operator.`

Reply:
SGTM